app/workers/field_tests/prune_old_experiments_worker.rb

```diff
@@ -0,0 +1,13 @@
+module FieldTests
+  class PruneOldExperimentsWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform
+      five_precent_membership_count = FieldTest::Membership.count / 20
+      memberships = FieldTest::Membership.first(five_precent_membership_count)
+      FieldTest::Event.where(field_test_membership_id: memberships.pluck(:id)).delete_all
```

Comment:
Hmm..is there a potential issue because one of our conversion targets takes place over multiple days? Might want to make sure we don't prune anybody who started an experiment less than 7 days ago (based on my reading of the historical data, the current oldest 5% of users are only 6 day old!

Reply:
Currently about 10% of our field tests were created _over_ 6 days ago, so this would remove _some_ of them, but keep a bunch more in the pool.

About 40% were created in the last 5 days, and less and less each day because returning users are not submitted to new field tests.

Given that we started this about 6.3 days ago, we'll get to add 5% back into the pool while still getting plenty of tests that will soon be 7+ days old, and in the future we'll have testers that stick around for longer and longer and longer.

Because 5% is so much less than 1/7, I think we should usually be adding new members faster than we prune them and we'll have many that go back more and more days going forward.